package.json

```diff
@@ -4,16 +4,26 @@
   "private": true,
   "dependencies": {
     "axios": "^0.26.1",
+    "bcryptjs": "^2.4.3",
+    "buffer": "^6.0.3",
+    "crypto": "^1.0.1",
+    "crypto-browserify": "^3.12.0",
     "dayjs": "^1.11.0",
+    "jsonwebtoken": "^8.5.1",
+    "jws": "^4.0.0",
     "jwt-decode": "^3.1.2",
     "jwt-encode": "^1.0.1",
-    "miragejs": "^0.1.43",
-    "mockman-js": "^1.1.5",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-router-dom": "^6.3.0",
+    "miragejs": "^0.1.41",
+    "mockman-js": "^1.0.5",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.2.2",
```

Comment:
what was the reason behind changing versions?

Reply:
I struggled to get the mock api to work which is why I tried changeing everything I guess at last it did work.